### ISSUE TITLE
Language Server - GNOME Builder Support

### ISSUE THREAD

**Issue by reesericci** [sveltejs/language-tools#1441]
### Description

I would like to be able to use the Svelte Language Server in GNOME Builder, so it can provide auto-completion and other fancy things. It's frustrating for me, especially since VS Code is borked on my machine, and doesn't integrate properly w/Fedora Silverblue.

### Proposed solution

A plugin for GNOME Builder that implements the language server via LSP.

### Alternatives

I don't know if there are any.

### Additional Information, eg. Screenshots

GNOME Builder is the official IDE for the GNOME Desktop (Linux).

**Comment by dummdidumm** [comment]
I don't think that we'll be able to/want to provide an integration ourselves with this IDE, but you are welcome to create one of your own and we will try to help out if you have any questions about integrating the language server.
Examples for integrations are the VS Code extension or [coc-svelte](https://github.com/coc-extensions/coc-svelte) which is also community-maintained.

**Comment by reesericci** [comment]
Can I put a bounty on this?
